2024-01-23 00:29:54 [main] INFO  c.a.t.t.mockTests.DbTests.EmptyTests -
                Started EmptyTests in 1.2 seconds (process running for 11.22)
2024-01-23 00:29:54 [main] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No orders found at all. Please contact your application team right away., uri=/api/owners-tools/orders/get-orders, timestamp=Tue, 23 Jan 2024 00:29:54 -0700}
2024-01-23 00:29:54 [main] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customers found at all. Please contact your application team right away., uri=/api/owners-tools/customers/get-customers, timestamp=Tue, 23 Jan 2024 00:29:54 -0700}
2024-01-23 00:29:54 [http-nio-auto-1-exec-6] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=You have chosen a menu item that does not exist., uri=/api/menu/id, timestamp=Tue, 23 Jan 2024 00:29:54 -0700}
2024-01-23 00:29:54 [http-nio-auto-1-exec-8] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Order item not updated. Verify order item is part of order., uri=/api/owners-tools/orders/update-order-item/654654-465465-555/88/2, timestamp=Tue, 23 Jan 2024 00:29:54 -0700}
2024-01-23 00:29:54 [http-nio-auto-1-exec-10] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=400 BAD_REQUEST, message=We were unable to process your request. Please contact us directly when trying to order more than 10 of any given item., uri=/api/owners-tools/orders/update-order-item/654654-465465-555/3/12, timestamp=Tue, 23 Jan 2024 00:29:54 -0700}
2024-01-23 00:29:54 [http-nio-auto-1-exec-6] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Order item not updated. Verify order exists., uri=/api/owners-tools/orders/update-order-item/77/3/2, timestamp=Tue, 23 Jan 2024 00:29:54 -0700}
2024-01-23 00:29:54 [http-nio-auto-1-exec-9] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No order found with that UID. Please verify and try again., uri=/api/owners-tools/orders/get-order-uid/orderUid, timestamp=Tue, 23 Jan 2024 00:29:54 -0700}
2024-01-23 00:29:55 [http-nio-auto-1-exec-1] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Can not delete order. Verify order exists., uri=/api/owners-tools/orders/delete-order/78, timestamp=Tue, 23 Jan 2024 00:29:55 -0700}
2024-01-23 00:29:55 [http-nio-auto-1-exec-5] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customer found by that name. Please check your spelling and formatting., uri=/api/owners-tools/customers/get-customer-name/name, timestamp=Tue, 23 Jan 2024 00:29:55 -0700}
2024-01-23 00:29:55 [http-nio-auto-1-exec-7] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Menu item can not be added to order. Verify order id., uri=/api/owners-tools/orders/add-to-order/78/2/2, timestamp=Tue, 23 Jan 2024 00:29:55 -0700}
2024-01-23 00:29:55 [http-nio-auto-1-exec-9] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Unable to find com.adamstraub.tonsoftacos.entities.MenuItem with id 66, uri=/api/owners-tools/orders/add-to-order/654654-465465-555/66/2, timestamp=Tue, 23 Jan 2024 00:29:55 -0700}
2024-01-23 00:29:55 [http-nio-auto-1-exec-7] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No order found with that UID. Please verify and try again., uri=/api/owners-tools/orders/get-order-uid/orderUid, timestamp=Tue, 23 Jan 2024 00:29:55 -0700}
2024-01-23 00:29:55 [main] INFO  c.a.t.t.s.o.o.GetOrdersByCustomerTests$testThatDoesNotPolluteTheApplicationContextUris -
                Starting GetOrdersByCustomerTests.testThatDoesNotPolluteTheApplicationContextUris using Java 17.0.6 with PID 119060 (started by brian in /media/brian/biggin/Documents/Tablet Backup/Documents/Courses/Java/personal projects/TonsOfTacosV1.2/back-end/tons-of-tacos)
2024-01-23 00:29:55 [main] DEBUG c.a.t.t.s.o.o.GetOrdersByCustomerTests$testThatDoesNotPolluteTheApplicationContextUris -
                Running with Spring Boot v3.0.1, Spring v6.0.3
2024-01-23 00:29:55 [main] INFO  c.a.t.t.s.o.o.GetOrdersByCustomerTests$testThatDoesNotPolluteTheApplicationContextUris -
                No active profile set, falling back to 1 default profile: "default"
2024-01-23 00:29:56 [main] DEBUG c.a.t.config.security.JwtAuthFilter -
                Filter 'jwtAuthFilter' configured for use
2024-01-23 00:29:56 [main] INFO  c.a.t.t.s.o.o.GetOrdersByCustomerTests$testThatDoesNotPolluteTheApplicationContextUris -
                Started GetOrdersByCustomerTests.testThatDoesNotPolluteTheApplicationContextUris in 1.031 seconds (process running for 13.832)
2024-01-23 00:29:57 [http-nio-auto-4-exec-4] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No orders for customer found., uri=/api/owners-tools/orders/get-order-customer/customer, timestamp=Tue, 23 Jan 2024 00:29:57 -0700}
2024-01-23 00:29:57 [http-nio-auto-4-exec-5] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=403 FORBIDDEN, message=Session expired., uri=/api/owners-tools/customers/delete-customer/1, timestamp=Tue, 23 Jan 2024 00:29:57 -0700}
2024-01-23 00:29:57 [http-nio-auto-4-exec-6] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=401 UNAUTHORIZED, message=User unauthorized., uri=/api/owners-tools/customers/delete-customer/1, timestamp=Tue, 23 Jan 2024 00:29:57 -0700}
2024-01-23 00:29:57 [http-nio-auto-1-exec-3] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=400 BAD_REQUEST, message=Customer name incorrectly formatted. Please consult the documentation., uri=/api/owners-tools/customers/edit-customer-name/gd34-igjr/dsfk%20jh!@, timestamp=Tue, 23 Jan 2024 00:29:57 -0700}
2024-01-23 00:29:57 [http-nio-auto-1-exec-5] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customer with that id found., uri=/api/owners-tools/customers/edit-customer-name/54/Gary%20Garrison, timestamp=Tue, 23 Jan 2024 00:29:57 -0700}
2024-01-23 00:29:57 [http-nio-auto-1-exec-7] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=400 BAD_REQUEST, message=Order can not be closed if order is not ready., uri=/api/owners-tools/orders/close-order/654654-465465-555, timestamp=Tue, 23 Jan 2024 00:29:57 -0700}
2024-01-23 00:29:57 [http-nio-auto-1-exec-5] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Order cannot be found and as such can not be closed. Please verify order id., uri=/api/owners-tools/orders/close-order/66, timestamp=Tue, 23 Jan 2024 00:29:57 -0700}
2024-01-23 00:29:57 [http-nio-auto-1-exec-7] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customer found with that UID. Please verify and try  again., uri=/api/owners-tools/customers/get-customer-uid/customerUid, timestamp=Tue, 23 Jan 2024 00:29:57 -0700}
2024-01-23 00:29:58 [http-nio-auto-1-exec-7] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Order does not exist. Please verify order id and try again., uri=/api/owners-tools/orders/order-ready/88, timestamp=Tue, 23 Jan 2024 00:29:58 -0700}
2024-01-23 00:29:58 [http-nio-auto-4-exec-1] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=400 BAD_REQUEST, message=Customer name incorrectly formatted. Please consult the documentation., uri=/api/order/checkout, timestamp=Tue, 23 Jan 2024 00:29:58 -0700}
2024-01-23 00:29:58 [http-nio-auto-4-exec-3] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=403 FORBIDDEN, message=Bad credentials, uri=/api/owners-tools/login, timestamp=Tue, 23 Jan 2024 00:29:58 -0700}
2024-01-23 00:29:58 [http-nio-auto-4-exec-4] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=403 FORBIDDEN, message=Bad credentials, uri=/api/owners-tools/login, timestamp=Tue, 23 Jan 2024 00:29:58 -0700}
2024-01-23 00:29:58 [main] INFO  c.a.t.t.TonsOfTacosApplicationTests -
                Starting TonsOfTacosApplicationTests using Java 17.0.6 with PID 119060 (started by brian in /media/brian/biggin/Documents/Tablet Backup/Documents/Courses/Java/personal projects/TonsOfTacosV1.2/back-end/tons-of-tacos)
2024-01-23 00:29:58 [main] DEBUG c.a.t.t.TonsOfTacosApplicationTests -
                Running with Spring Boot v3.0.1, Spring v6.0.3
2024-01-23 00:29:58 [main] INFO  c.a.t.t.TonsOfTacosApplicationTests -
                No active profile set, falling back to 1 default profile: "default"
2024-01-23 00:29:59 [main] INFO  c.a.t.t.TonsOfTacosApplicationTests -
                Started TonsOfTacosApplicationTests in 0.875 seconds (process running for 16.875)
2024-01-23 00:29:59 [http-nio-auto-1-exec-2] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customer with that id found., uri=/api/owners-tools/customers/edit-customer-email/13/larry@larbo.com, timestamp=Tue, 23 Jan 2024 00:29:59 -0700}
2024-01-23 00:30:00 [http-nio-auto-1-exec-8] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=400 BAD_REQUEST, message=Email does not match formatting requirements, please consult the docs., uri=/api/owners-tools/customers/edit-customer-email/gd34-igjr/larrylarbo.com, timestamp=Tue, 23 Jan 2024 00:30:00 -0700}
2024-01-23 00:56:14 [main] INFO  c.a.t.t.s.u.GetMenuCategoriesTests$testThatDoesNotPolluteTheApplicationContextUris -
                Starting GetMenuCategoriesTests.testThatDoesNotPolluteTheApplicationContextUris using Java 17.0.6 with PID 120296 (started by brian in /media/brian/biggin/Documents/Tablet Backup/Documents/Courses/Java/personal projects/TonsOfTacosV1.2/back-end/tons-of-tacos)
2024-01-23 00:56:14 [main] DEBUG c.a.t.t.s.u.GetMenuCategoriesTests$testThatDoesNotPolluteTheApplicationContextUris -
                Running with Spring Boot v3.0.1, Spring v6.0.3
2024-01-23 00:56:14 [main] INFO  c.a.t.t.s.u.GetMenuCategoriesTests$testThatDoesNotPolluteTheApplicationContextUris -
                No active profile set, falling back to 1 default profile: "default"
